### PR TITLE
UPGRADE FOUNDATION ON MANAGER: As a PO, I want the foundation gem and styles upgraded so that I know we are up to date

### PR DESCRIPTION
**Pull request by Reinkaos**
Acceptance Criteria

Foundation 6 installed on manager app

### CODE REVIEW

File: app/assets/stylesheets/application.css.scss [hunk 1]
```diff
@@ -40,3 +39,20 @@
...
+
+.pagination-previous a::before, .pagination-previous.disabled::before,
+.pagination-next a::after, .pagination-next.disabled::after {
+  display: none;
+}
+
+.pagination-previous.disabled, .pagination-next.disabled {
+  a {
+    padding: 0.1875rem 0.625rem;
+    color: #cacaca;
```
Comment (yarhtut) [inline app/assets/stylesheets/application.css.scss]: $variable? only if you have bro

File: app/views/abstract_jobs/index.html.erb [hunk 2]
```diff
@@ -8,19 +8,17 @@ Supplejack was created by DigitalNZ at the National Library of NZ and the Depart
...
-    <dl class="tabs pill">
-      <% ['active', 'finished', 'failed', 'stopped'].each do |status| %>
-        <%= content_tag(:dd, link_to(t("harvest_jobs.#{status}"), environment_abstract_jobs_path(params[:environment], status: status, parser: params[:parser])), class: ("active" if params[:status] == status)) %>
-      <% end %>
-    </dl>
-  </div>
-</div>
+<% ['active', 'finished', 'failed', 'stopped'].each do |status| %>
+  <% if params[:status] == status %>
+    <%= link_to(t("harvest_jobs.#{status}"), environment_abstract_jobs_path(params[:environment], status: status, parser: params[:parser]), class: 'button primary') %>
```
Comment (yarhtut) [inline app/views/abstract_jobs/index.html.erb]: we may not need `primary` in button class.

File: app/assets/stylesheets/foundation_and_overrides.scss [hunk 3]
```diff
@@ -1,265 +1,56 @@
...
-   @extend .radius;
-}
+// If you'd like to include motion-ui the foundation-rails gem comes prepackaged with it, uncomment the 3 @imports, if you are not using the gem you need to install the motion-ui sass package.
+//
+// @import 'motion-ui/motion-ui';
+
+// We include everything by default.  To slim your CSS, remove components you don't use.
+
+@include foundation-global-styles;
+@include foundation-xy-grid-classes;
```
Comment (yarhtut) [inline app/assets/stylesheets/foundation_and_overrides.scss]: oh nice you using `xy-grid`

File: app/views/home/_stats.html.erb [hunk 4]
```diff
@@ -1,37 +1,36 @@
...
+third party components that are licensed under the MIT license or otherwise publicly available.
+See https://github.com/DigitalNZ/supplejack_manager for details.
 
-Supplejack was created by DigitalNZ at the National Library of NZ and the Department of Internal Affairs. 
+Supplejack was created by DigitalNZ at the National Library of NZ and the Department of Internal Affairs.
 http://digitalnz.org/supplejack
 %>
 
-<div class="row">
+<div class="grid-x">
```
Comment (yarhtut) [inline app/views/home/_stats.html.erb]: 👍

File: app/views/previewers/create.js.erb [hunk 5]
```diff
@@ -12,4 +12,5 @@ http://digitalnz.org/supplejack
 %>
 
 $("#preview-area").html('<%= j render "previewer" %>');
-PreviewJobsPoller.poll();
\ No newline at end of file
+$(document).foundation();
```
Comment (yarhtut) [inline app/views/previewers/create.js.erb]: `foundation() ` load happen hear. I saw that in your application.js. Maybe it is ok.

File: app/assets/stylesheets/application.css.scss (hunk 1 above)
Comment (Reinkaos) [reply app/assets/stylesheets/application.css.scss]: I will :)

File: app/views/previewers/create.js.erb (hunk 5 above)
Comment (Reinkaos) [reply app/views/previewers/create.js.erb]: Yeah I need this because this document gets ajaxed in and foundation is not yet bound.

**Comment by DanHenton** [comment]
👍

**Pull request merged**